src/less/toast/base/toast.less

```diff
@@ -0,0 +1,97 @@
+:root {
+    --ebay-skin-toast-spacing: 24px;
+}
+
+.toast__mobile {
+    --ebay-skin-toast-spacing: 16px;
+}
+
+.transition() {
+    transition:
+        opacity 0.2s cubic-bezier(0.21, 0.31, 1, 1.22) 0s,
+        transform 0.2s cubic-bezier(0.21, 0.31, 1, 1.22) 0s;
+}
+
+.toast {
+    background-color: @toast-box-color;
+    bottom: 16px;
+    color: @white-color;
+    left: 16px;
+    max-height: 40vh;
+    max-width: 480px;
+    min-width: 320px;
+    position: fixed;
+    z-index: 1;
+
+    &.toast--transition {
+        display: block;
+        transform: translateY(110%);
+    }
+}
+
+.toast.toast__mobile {
```

Comment:
Shouldn't need `.toast` at the beginning, just label it `.toast__mobile`
Same with all others in this.

Reply:
This is done so that we have the css variable re-configured to take a different value in mobile layouts.

Current CSS linting complains about css variables to not be declared along side with styles.
```
.toast__mobile {
    --ebay-skin-toast-spacing: 16px;
}
...
.toast.toast__mobile {
    ...
}
```

Reply:
So i get that. However i would also say that we shouldn’t try to add extra rules to get by the liner

A couple of solutions:
1) use media queries to show mobile layout (again look at dialog how it uses the media queries). That way you can have both the variable declared in the top level (like in the media query) and then add mobile specific styles (either inside the media query and get rid of the mobile selector or outside media query and use selector. I like responsive design so i prefer using the media query)
2) put the variable in another file like a variables file and import it that way.

Reply:
Regarding CSS custom properties. I have not yet documented how we intend to use them, but unofficially at least for now it is colors only (like in the switch module). Over time we might open it up to spacing, but not just yet.

Reply:
Though in general, I do agree to not add extra rules to get by the liner, this is a special case where the extra rule is pretty simple and avoiding it by discovering workarounds at that time was expensive until experts like Ian and you chime in in PRs :)

**Regarding media queries**
Based on my understanding working in Skin and review comments, we are supposed to create classes and leave the flexibility for when to add media queries to clients.

**Regarding Separate File**
The linting problem is in the generated CSS files so I did not expect this solution would work. _Glad that it worked! Thanks_